src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/RazorProjectService.cs

```diff
@@ -254,10 +252,10 @@ public ProjectKey AddProject(string filePath, string intermediateOutputPath, Raz
 
         _projectSnapshotManagerAccessor.Instance.ProjectWorkspaceStateChanged(project.Key, projectWorkspaceState);
 
-        var currentHostProject = project.HostProject;
```

Comment:
ℹ️ It turns out that accessing `ProjectSnapshot.HostProject` was unnecessary here and the code can just fallback to `IProjectSnapshot`.